dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -258,12 +258,15 @@ private Tool refreshContainer(final long containerId, final long userId) {
     @Timed
     @UnitOfWork
     @Path("/{containerId}")
-    @ApiOperation(value = "Get a registered repo", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Tool.class)
+    @ApiOperation(value = "Get a registered repo", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Tool.class, notes = "Also loads the users")
     public Tool getContainer(@ApiParam(hidden = true) @Auth User user,
             @ApiParam(value = "Tool ID", required = true) @PathParam("containerId") Long containerId) {
         Tool c = toolDAO.findById(containerId);
         checkEntry(c);
         checkUser(user, c);
+        
+        // This somehow forces users to get loaded
```

Comment:
Did you end up discovering why this is necessary and how this works given that it seems like checkUser should already iterate through the users collection?

Reply:
something was wrong previously with the lazy loading, you can remove the checkEntry and checkUser and it would still return the users.